.circleci/config.yml

```diff
@@ -255,3 +267,12 @@ workflows:
           requires:
             - test_unit
             - test_karma
+
+      - deploy:
+          requires:
+            - test_integration
+            - test_integration_compat
+          filters:
+            tags:
```

Comment:
Question: Do you think it makes sense to prevent this step from running if the branch is not a release branch?

Reply:
Including both the `tags` and `branches` filters results in an `||` condition instead of an `&&` condition so I put the restriction into `publish.js`.